examples/demo/src/index.ts

```diff
@@ -137,3 +184,19 @@ const ed = editor.create(document.getElementById('editor')!, {
 window.addEventListener('hashchange', () => {
   ed.setModel(getModel());
 });
+
+tailwindrcModel.onDidChangeContent(() => {
+  let tailwindConfig: unknown;
+  try {
+    tailwindConfig = parse(tailwindrcModel.getValue());
+  } catch {
+    return;
+  }
+  if (typeof tailwindConfig !== 'object') {
+    return;
+  }
+  if (tailwindConfig == null) {
```

Comment:
why can we not use `===` hat the same ts problem yesterday ... i dislike `==` ^^

Reply:
The ESLint rule [`eqeqeq`](https://eslint.org/docs/rules/eqeqeq#options) rule enforces the use of `===` over `==`, but it has been configured to actually enforce `==` for null checks. The rationale is that 99 out of 100 times, the difference between `null` and `undefined` doesn’t matter.

I setup a very strict ESLint config, because I like to not think about such stylistic issues and prefer to have a tool validate and autofix it for me. :)

Reply:
I’m also ok with checking falsiness if you prefer that

```suggestion
  if (!tailwindConfig) {
```

Reply:
thanks for the explanation ^^ i was not really sure what was type casted. youre right `the difference between null and undefined doesn’t matter`

coming from php those lose type checks were a dorn in my eye (`==` makes ANYTHING match in php `'false' == false`)

so this can stay ^^ seems like a good oll `isset()` check ;)